script/upgrade.rb

```diff
@@ -0,0 +1,62 @@
+class String
+def green; "\033[32m#{self}\033[0m" end
+def red;   "\033[31m#{self}\033[0m" end
+end
+
+puts("How to upgrade guide".green)
+puts("Check latest releases at https://github.com/churchio/onebody/releases and select version to upgrade.")
+puts("1: upgrade from 3.3.0 to 3.4.0\n")
+puts("2: upgrade from 3.2.0 to 3.3.0\n")
+puts("3: upgrade from 3.1.0 to 3.2.0\n")
+puts("4: upgrade from 3.0.0 to 3.1.0\n")
+puts("Please select version you would like to upgrade(1/2/3/4): ")
+input = gets.chomp
+if input == '1' || input == "1." then
```

Comment:
Never use `then` for multi-line `if`.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.